lib/modules/manager/terraform/extractors/resources/helm-release.ts

```diff
@@ -25,9 +26,18 @@ export class HelmReleaseExtractor extends DependencyExtractor {
           depName: helmRelease.chart,
           datasource: HelmDatasource.id,
         };
+
+        // For oci charts, we remove the oci:// and use the docker datasource
+        const isOciChart = checkIfChartIsOCI(helmRelease.chart);
+        if (isOciChart) {
+          dep.depName = helmRelease.chart.replace(ociRegex, '');
+          dep.datasource = DockerDatasource.id;
```

Comment:
move this after the first if condition below, so the chart property is validated first.

that will simplify all conditions.